spec/controllers/idv/cac_controller_spec.rb

```diff
@@ -28,6 +28,12 @@
     stub_sign_in(user)
   end
 
+  describe 'before_actions' do
+    it 'includes before_actions from IdvSession' do
+      expect(subject).to have_actions(:sp_context_needed?)
```

Comment:
WDYT of using `controller` instead of subject?

```suggestion
      expect(controller).to have_actions(:sp_context_needed?)
```